fedimint-server/src/consensus/engine.rs

```diff
@@ -387,6 +389,8 @@ impl ConsensusEngine {
 
         let mut signatures = BTreeMap::new();
 
+        let items_dump = tokio::sync::OnceCell::new();
```

Comment:
Why `OnceCell` and not a simple mutable boolean? Does the `&mut bool` cause lifetime issues inside the `select!`?

Reply:
No. I just though `OnceCell` expresses better that we want it to happen at max once. :D @elsirion

Reply:
So be it :man_shrugging: probably not hurting anything, just slightly unconventional I guess (and it saves a mut var!)